src/components/EmptyMailboxSection.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-	<div id="emptycontent">
-		<slot> {{ t('mail', 'Messages marked as important will show up here') }} </slot>
```

Comment:
I believe this file might be obsolete now

Reply:
I don't think that is the case. `EmptyMailboxSection` renders when `is-priority-inbox=true`, which is only done for the two priority inbox sections in `MailboxThread`. And we no longer show the *important* mailbox when it's empty, only *Other*.

So whenever *Other* is empty, we show a display-none'd `EmptyMailboxSection`.

This means we have to undo the display none and rectify `Messages marked as important will show up here` to make sense for *Others*. It was a wrong assumption that the component would only ever show for important emails.

Reply:
when i switch between mailboxes, this is shown
![Screenshot from 2022-10-17 16-18-39](https://user-images.githubusercontent.com/12728974/196201499-e61a7d89-a576-4745-96e9-4462c36afcd4.png)

Reply:
exactly, that was my problem since the beginning i cannot hide the label

Reply:
Test again. I accidentally undid one of my changes when I committed. Fixed now.

Reply:
yes, all looks good now